tests/test_tifffile.py

```diff
@@ -144,6 +144,34 @@ def test_imagej_hyperstack(tmp_path):
     assert img.shape == (15, 2, 180, 183)
 
 
+def test_resolution_metadata(tmp_path):
+    data = np.zeros((200, 100), dtype=np.uint8)
+
+    # test rational DPI (reduced)
+    dpi = (100, 200)
+
+    writer = imageio.get_writer(tmp_path / "test.tif")
+    writer.append_data(data, dict(resolution=dpi))
+    writer.close()
+
+    read_image = iio.imread(tmp_path / "test.tif")
+
+    assert read_image.meta["resolution"] == (*dpi, "INCH")
+    assert read_image.meta["resolution_unit"] == 2
```

Comment:
I'm confused, here the numbers are not rationals?

Reply:
True. Yesterday, it seemed like a good idea to reduce a rational like `(200, 1)` to `200`, i.e., if the denominator is 1, drop it. Today, I'm not so sure anymore how good this idea actually is, considering that it makes the return type inconsistent.

What do you think?

Reply:
I think we should at least be consistent. So either we always return `(int, int)` or we always return `float`. My gut says floats but I also see the value in just passing what tifffile returns. Your call :D

Reply:
I cannot imagine that the precision matters *that* much. I think I'm leaning towards floats too; the int-tuple is confusing.

Reply:
lets go with `float` then. I made the necessary changes.